### PR TITLE
Show clear feedback to admin on polls being synced

### PR DESCRIPTION
**Pull request by norkans7**
This will make it more clear for admin that keeps creating more polls and expect them to be displayed on the public site right away

### CODE REVIEW

**Comment by norkans7** [comment]
U-Report 2020-05-05 13-49-03

File: locale/ar/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -820,6 +820,36 @@ msgstr ""
 msgid "Backends"
 msgstr ""
 
+#, python-format
+msgid ""
+"\n"
+"        New polls are not displayed to the public site until we have synced results to display.\n"
+"        <br>\n"
+"        <b>\n"
+"          We currently have %(counter)s poll syncing in the background.\n"
```
Comment (nicpottier) [inline locale/ar/LC_MESSAGES/django.po:829]: There is/are currently __ polls syncing in the background. (use plural)

File: locale/ar/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -820,6 +820,36 @@ msgstr ""
...
+"        </b>\n"
+"        <br>\n"
+"        This message will dissappear when all polls synced and have results to display\n"
```
Comment (nicpottier) [inline locale/ar/LC_MESSAGES/django.po:832]: Don't think we need this.

File: locale/ar/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -820,6 +820,36 @@ msgstr ""
 msgid "Backends"
 msgstr ""
 
+#, python-format
+msgid ""
+"\n"
+"        New polls are not displayed to the public site until we have synced results to display.\n"
```
Comment (nicpottier) [inline locale/ar/LC_MESSAGES/django.po:826]: Maybe just one message:

`There is/are currently _ new poll(s) syncing. Polls are displayed on the public site once they have completely synced their results.`

**Pull request merged**